tests/Unit/Domain/Test_NormalDotFlux.cpp

```diff
@@ -145,10 +171,176 @@ void check() {
 }  // namespace
 
 SPECTRE_TEST_CASE("Unit.Evolution.NormalDotFluxCompute", "[Unit][Evolution]") {
-  check<1, Frame::Inertial>();
-  check<1, Frame::Grid>();
-  check<2, Frame::Inertial>();
-  check<2, Frame::Grid>();
-  check<3, Frame::Inertial>();
-  check<3, Frame::Grid>();
+  {
+    INFO("Explicit values");
+    const size_t npts = 5;
+    const DataVector zero(npts, 0.0);
+    const DataVector one(npts, 1.0);
+    const DataVector two(npts, 2.0);
+    const DataVector three(npts, 3.0);
+    const DataVector four(npts, 4.0);
+    const DataVector five(npts, 5.0);
+    const DataVector six(npts, 6.0);
+    const DataVector seven(npts, 7.0);
+    const DataVector eight(npts, 8.0);
+    const DataVector nine(npts, 9.0);
+    const DataVector ten(npts, 10.0);
+    const DataVector eleven(npts, 11.0);
+    const DataVector twelve(npts, 12.0);
+    const DataVector fifteen(npts, 15.0);
+    const DataVector eighteen(npts, 18.0);
+
+    check_normal_dot_flux(tnsr::i<DataVector, 1>{{{one}}},
+                          tnsr::I<DataVector, 1>{{{two}}},
+                          Scalar<DataVector>{two});
+    check_normal_dot_flux(tnsr::i<DataVector, 2>{{{one, two}}},
+                          tnsr::I<DataVector, 2>{{{three, four}}},
+                          Scalar<DataVector>{eleven});
+    check_normal_dot_flux(tnsr::i<DataVector, 3>{{{one, two, three}}},
+                          tnsr::I<DataVector, 3>{{{-four, -two, three}}},
+                          Scalar<DataVector>{one});
+    check_normal_dot_flux(tnsr::i<DataVector, 3>{{{one, two, three}}},
```

Comment:
isn't this a repeat?

Reply:
Yep, will remove.

Do you think I should actually keep these specific tests around or is the Pypp Test enough? I just computed these numbers in Python as well..